src/pyhf/readxml.py

```diff
@@ -343,20 +367,41 @@ def dedupe_parameters(parameters):
     return list({v['name']: v for v in parameters}.values())
 
 
-def parse(configfile, rootdir, track_progress=False, validation_as_error=True):
+def parse(
+    configfile: PathOrStr | T.IO[bytes] | T.IO[str],
+    rootdir: PathOrStr,
+    mounts: T.Optional[MountPathType] = None,
+    track_progress: bool = False,
+    validation_as_error: bool = True,
+):
+    """
+    Parse the configfile with respect to the rootdir.
+
+    Args:
+        mounts (:obj:`None` or :obj:`list` of 2-:obj:`tuple` of :class:`pathlib.Path` objects): The first field is the local path to where files are located, the second field is the path where the file or directory are saved in the XML configuration. This is similar in spirit to docker. Default is ``None``.
```

Comment:
Uniformed question: Why are we only adding docstrings for the last 3 args? Why do we not also add them for `configfile` and `rootdir`?

Reply:
added